deployment/ccip/changeset/state.go

```diff
@@ -750,3 +771,33 @@ func LoadChainState(ctx context.Context, chain deployment.Chain, addresses map[s
 	}
 	return state, nil
 }
+
+func (s CCIPOnChainState) ValidateOffRamp(chainSelector uint64) error {
+	family, err := chain_selectors.GetSelectorFamily(chainSelector)
+	if err != nil {
+		return err
+	}
+	switch family {
+	case chain_selectors.FamilyEVM:
+		chainState, exists := s.Chains[chainSelector]
+		if !exists {
+			return fmt.Errorf("chain %d does not exist", chainSelector)
+		}
+		if chainState.OffRamp == nil {
+			// should not be possible, but a defensive check.
+			return errors.New("OffRamp contract does not exist")
```

Comment:
Some more ctx in the error might be useful


```suggestion
			return fmt.Errorf("OffRamp contract does not exist on chain %d", chainSelector)
```